src/OpenTelemetry.AutoInstrumentation/Configuration/MeterSettings.cs

```diff
@@ -0,0 +1,159 @@
+// <copyright file="MeterSettings.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenTelemetry.AutoInstrumentation.Configuration
+{
+    /// <summary>
+    /// Meter Settings
+    /// </summary>
+    public class MeterSettings : Settings
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MeterSettings"/> class
+        /// using the specified <see cref="IConfigurationSource"/> to initialize values.
+        /// </summary>
+        /// <param name="source">The <see cref="IConfigurationSource"/> to use when retrieving configuration values.</param>
+        private MeterSettings(IConfigurationSource source)
+            : base(source)
+        {
+            MetricExporter = ParseMetricExporter(source);
+            ConsoleExporterEnabled = source.GetBool(ConfigurationKeys.Metrics.ConsoleExporterEnabled) ?? false;
+
+            var instrumentations = new Dictionary<string, MeterInstrumentation>();
+            var enabledInstrumentations = source.GetString(ConfigurationKeys.Metrics.Instrumentations);
+            if (enabledInstrumentations != null)
+            {
+                foreach (var instrumentation in enabledInstrumentations.Split(separator: ','))
```

Comment:
This pattern doesn't come from this PR, but, now we have the typical separator, comma, and the one for plugins hard coded in many places. It seems reasonable to consolidate them. That said it will be better to do it in a separate PR.

Reply:
Created issue #678 to track this effort.